scim/data_account_group.go

```diff
@@ -0,0 +1,93 @@
+package scim
+
+import (
+	"context"
+	"sort"
+	"strings"
+
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+// DataSourceGroup returns information about group specified by display name
+func DataSourceAccountGroup() *schema.Resource {
+	type entity struct {
+		DisplayName       string   `json:"display_name"`
+		Recursive         bool     `json:"recursive,omitempty"`
+		Members           []string `json:"members,omitempty" tf:"slice_set,computed"`
+		Users             []string `json:"users,omitempty" tf:"slice_set,computed"`
+		ServicePrincipals []string `json:"service_principals,omitempty" tf:"slice_set,computed"`
+		ChildGroups       []string `json:"child_groups,omitempty" tf:"slice_set,computed"`
+		Groups            []string `json:"groups,omitempty" tf:"slice_set,computed"`
+		Roles             []string `json:"roles,omitempty" tf:"slice_set,computed"`
+		ExternalID        string   `json:"external_id,omitempty" tf:"computed"`
+	}
+
+	s := common.StructToSchema(entity{}, func(
+		s map[string]*schema.Schema) map[string]*schema.Schema {
+		// nolint once SDKv2 has Diagnostics-returning validators, change
+		s["display_name"].ValidateFunc = validation.StringIsNotEmpty
+		s["recursive"].Default = true
+		s["members"].Deprecated = "Please use `users`, `service_principals`, and `child_groups` instead"
```

Comment:
Why do we need to add it just to mark as deprecated?

Reply:
our SCIM API returns Members, for the data sources we are splitting them into users, sp & groups